src/friktion/infos.ts

```diff
@@ -340,4 +353,51 @@ export class VaultInfoWrapper {
   getAPY(): number {
     return this.vaultInfo.snapshotInfo ? this.vaultInfo.snapshotInfo.apy : 0;
   }
+  async getSharePrice(
+    connection: Connection,
+    round?: number,
+    claimDeposit?: boolean,
```

Comment:
What is the scenario for fetching sharePrice for neither deposit or withdrawal?? Can replace these 2 boolean params with 1 boolean only?

Reply:
It’s for knowing a round’s pending deposit or pending withdrawal price, what’s the naming will be if there’s only one boolean.
How about `trueFroDepositFalseForWithdrawal`